src/data/faq.json

```diff
@@ -492,7 +492,7 @@
                         "anchor": "where_to_get",
                         "active": true,
                         "textblock": [
-                            "Since the app is available, you can download it from the Apple App Store or Google Play Store."
+                            "Since the app is available, you can download it from the <a href='https://apps.apple.com/de/app/corona-warn-app/id1512595757' target='_blank' rel='noopener noreferrer'>Apple App Store</a> or <a href='https://play.google.com/store/apps/details?id=de.rki.coronawarnapp' target='_blank' rel='noopener noreferrer'>Google Play Store</a>."

```

Comment:
If you want to link specifically to the English language entries, the links are:

- https://apps.apple.com/de/app/corona-warn-app/id1512595757?l=en
- https://play.google.com/store/apps/details?id=de.rki.coronawarnapp&hl=en

Reply:
fixed the wording

For the language, it's probably better if the website identifies the users' language via User Agent. Because I think we have a turkish language store version.